test/verify/check-machines

```diff
@@ -483,6 +483,80 @@ class TestMachines(MachineCase):
 
         b.wait_in_text("#vm-subVmTest1-network-2-state span", "up")
 
+    def testVCPU(self):
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_present("tbody tr th")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+        b.click("tbody tr th") # click on the row header
+
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
+
+        b.wait_present("#vm-subVmTest1-vcpus-count") # wait for the tab
+        b.click("#vm-subVmTest1-vcpus-count") # open VCPU modal detail window
+
+        b.wait_present(".vcpu-detail-modal-table")
+        # Test basic vCPU properties
+        b.wait_val("#machines-vcpu-count-field", "1")
+        b.wait_val("#machines-vcpu-max-field", "1")
+
```

Comment:
Here it would make sense to check that the "only after reboot" info bar is shown.

Reply:
It already check it, but little bit below.

Reply:
I see, it's in the  section which is again testing a running VM. So perhaps better to move that here, and use the second section for testing changes while the VM is shut down.